06_Prepare_MICS6_01_Batch_final.do

```diff
@@ -179,6 +179,9 @@ clear
 		if inlist(country_name,"Iraq2017","KyrgyzRepublic2018","Mongolia2018","Suriname2018","Tunisia2018","Lesotho2018","Georgia2018","Bangladesh2019","Montenegro2018") {
 			gen c_ITN = .
 		}
+		if inlist(country_name,"Belarus2019") {
```

Comment:
@wengxyu1030 Thank you for your comments. When I tried to add the country name in line 179, it reported an error: expression too long, so I had to start a new command.  Actually, if I replace line 179 and 180 with gen c_ITN=., this problem can be solved. But I am not sure the purpose of generating c_ITN in this way (maybe it's written to remind us that we should check for the presence of data used to generate c_ITN when we work on each survey, rather than simply generating missing values?), so I left it as it was and wrote a new command. Do you need me to replace it with gen c_ITN=. ?

Reply:
Dear @jianingwwww ,

Thanks for your thoughtful clarification. I like your suggestion of replacing it just with `gen c_ITN=.` instead! I will quote this in the template refinement issue and talk with the team. 

For now please keep it as it is, no further reaction is needed. I will merge this pull request accordingly.

Regards, 
Aline